far2l/ctrlobj.cpp

```diff
@@ -55,7 +55,15 @@ ControlObject *CtrlObject;
 
 ControlObject::ControlObject():
 	FPanels(0),
-	CmdLine(0)
+	CmdLine(0),
```

Comment:
dont see value in this initializers as all stuff inited just here in c-tor, better do this in class declaration - this will at least make sense for potentially other constructors